media-libs/libgroove/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
Usually I open a bug with a new ebuild and the dev who commits it becomes the maintainer. If it's different now, no problem, I'm just not used to the new workflow.